atc/api/accessor/accessor.go

```diff
@@ -42,19 +42,23 @@ func (a *access) Claims() jwt.MapClaims {
 }
 
 func (a *access) IsAuthorized(team string) bool {
+	if a.IsAdmin() {
+		return true
+	}
 	for teamName, teamRoles := range a.TeamRoles() {
-		if teamName == team {
-			for _, teamRole := range teamRoles {
-				if a.HasPermission(teamRole) {
-					return true
-				}
+		if teamName != team {
+			continue
+		}
+		for _, teamRole := range teamRoles {
+			if a.hasPermission(teamRole) {
+				return true
 			}
 		}
 	}
 	return false
 }
 
-func (a *access) HasPermission(role string) bool {
+func (a *access) hasPermission(role string) bool {
```

Comment:
Just curious, do we have to change this to lower case? Is there some reason to do so?

Reply:
IIRC the reasoning behind the change was that there were no uses for `HasPermissions` outside the scope of the package